### PR TITLE
feat(Hypernative): Settings banner for activated HN Guardian

### PR DESCRIPTION
**Pull request by vmosharova**
## What it solves

Implements a new activated settings banner for Hypernative Guardian that displays on the Settings → Security page when the Hypernative Guardian is active.

Resolves: [COR-920](https://linear.app/safe-global/issue/COR-920/add-hypernative-guardian-card-in-security-settings)

## How this PR fixes it

Added a new "activated" banner `HnActivatedSettingsBanner` on the Settings page that appears when Hypernative Guardian is active, replacing the promotional banner.

### Conditional Rendering Logic
The Settings page (`SecurityLogin`) shows banners with the following logic:
1. `HnActivatedBannerForSettings` (with the Hypernative link) — shows when:
   - Hypernative Guardian is active
   - Hypernative feature is enabled
   - User is a Safe owner
2. `HnBannerForSettings` (HN promotion banner) — shows when:
   -  Hypernative Guardian is **not** active
   - Hypernative feature is enabled
   - User is a Safe owner
   - Other visibility conditions are met (balance, targeted Safe, etc.)

Mutual exclusivity: Only one banner shows at a time. The HOCs (`withHnFeature`, `withOwnerCheck`, `withGuardCheck`, `withHnBannerConditions`) handle visibility, and `useBannerVisibility` ensures the promo banner doesn't show when the guard is active.

### Implementation Details
- HOCs applied to `HnActivatedBannerForSettings`:
  - `withOwnerCheck` — only for Safe owners
  - `withGuardCheck` — only when guard is active
  - `withHnFeature` — only when feature is enabled
- Styling: CSS modules with theme variables
- Storybook: Story file for component documentation
- Tests covering conditional rendering, mutual exclusivity, edge cases, and loading states

image

## How to test it

1. Navigate to Settings → Security page
2. When Hypernative Guardian is activated, verify the banner appears at the top of the page

- The Storybook story: `Features/Hypernative/HnActivatedSettingsBanner`

## Checklist

- [ ] I've tested the branch on mobile 📱
- [ ] I've documented how it affects the analytics (if at all) 📊
- [x] I've written a unit/e2e test for it (if applicable) 🧑‍💻

---

## CLA signature

With the submission of this Pull Request, I confirm that I have read and agree to the terms of the [Contributor License Agreement](https://safe.global/cla).

### CODE REVIEW

**Comment by vmosharova** [comment]
@codex review

**Comment by vmosharova** [comment]
@claude review

File: apps/web/src/features/hypernative/components/HnActivatedSettingsBanner/HnActivatedSettingsBanner.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  gap: '8px',
+  backgroundColor: 'transparent',
+  textTransform: 'none',
+  inlineSize: 'fit-content',
+  '&:hover': {
+    borderColor: 'var(--color-primary-main, #121312)',
+    backgroundColor: 'transparent',
+  },
+  paddingInline: '16px',
+}
```
Comment (katspaugh) [inline apps/web/src/features/hypernative/components/HnActivatedSettingsBanner/HnActivatedSettingsBanner.tsx]: This claude be moved to the CSS module.

File: apps/web/src/styles/vars.css [hunk 2]
```diff
@@ -42,6 +42,7 @@
   --color-static-light: #636669;
   --color-static-primary: #ffffff;
   --color-static-text-secondary: #a1a3a7;
+  --color-text-brand: #12ff80;
```
Comment (katspaugh) [inline apps/web/src/styles/vars.css]: If it's the same for light and dark mode, we should probably call it `--color-static-text-brand`.

File: apps/web/src/features/hypernative/components/HnActivatedSettingsBanner/HnActivatedSettingsBanner.tsx (hunk 1 above)
Comment (vmosharova) [reply apps/web/src/features/hypernative/components/HnActivatedSettingsBanner/HnActivatedSettingsBanner.tsx]: Resolved

File: apps/web/src/styles/vars.css (hunk 2 above)
Comment (vmosharova) [reply apps/web/src/styles/vars.css]: Resolved

File: apps/web/src/features/hypernative/components/HnActivatedSettingsBanner/styles.module.css [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+.ctaButton {
+  border: 2px solid;
+  border-color: var(--color-primary-main, #121312);
+  border-radius: 6px;
+  padding: 6px 16px;
+  gap: 8px;
+  background-color: transparent;
+  text-transform: none;
+  inline-size: fit-content;
+  padding-inline: 16px;
```
[the remark this answers is not in the document]
Comment (vmosharova) [reply apps/web/src/features/hypernative/components/HnActivatedSettingsBanner/styles.module.css]: Resolved
